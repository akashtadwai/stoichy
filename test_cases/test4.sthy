def main()
{

string eq ;
element S ( 8,16.0,2) ;
element Br ( 16, 32.0 , 2 ) ;
element Mn ( 25 , 55.0 , 2 ) ;
element O (8 , 16.0 , 2) ;
element K (19 , 39.0 , 1) ;
element H (1 , 2.0 , 1) ;
molecule KBr (K,Br) ;
molecule KMnO4 (K, Mn, O, O, O, O) ;
molecule H2SO4 (H, H, S , O, O, O, O) ;
//eq = balance [KBr,KMnO4,H2SO4 --> Br2,MnSO4,K2SO4,H2O] ;
eq=balance[C2H6, O2 --> CO2,H2O];
print eq;

}
