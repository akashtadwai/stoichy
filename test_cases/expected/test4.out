 Printing without using concatenation
Hello, world!
 Printing by using concatenation
Hello, world!
